Source/Microsoft.Teams.Apps.FAQPlusPlus.Common/Providers/SearchServiceDataProvider.cs

```diff
@@ -0,0 +1,110 @@
+﻿// <copyright file="SearchServiceDataProvider.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+
+namespace Microsoft.Teams.Apps.FAQPlusPlus.Common.Providers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker.Models;
+    using Microsoft.Teams.Apps.FAQPlusPlus.Common.Models;
+    using Microsoft.WindowsAzure.Storage;
+    using Microsoft.WindowsAzure.Storage.Blob;
+    using Newtonsoft.Json;
+
+    /// <summary>
+    /// Azure search service blob storage data provider.
+    /// </summary>
+    public class SearchServiceDataProvider : ISearchServiceDataProvider
+    {
+        /// <summary>
+        /// File name storing JSON structured QnA records.
+        /// </summary>
+        private const string FaqPlusQnAFile = "/faqplusqnadata.json";
+
+        private readonly IQnaServiceProvider qnaServiceProvider;
+        private readonly string azureWebJobsStorage;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SearchServiceDataProvider"/> class.
+        /// </summary>
+        /// <param name="qnaServiceProvider">question and answer ServiceProvider.</param>
+        /// <param name="azureWebJobsStorage">Azure web job storage.</param>
```

Comment:
This isn't azureWebJobsStorage. Maybe storageConnectionString

Reply:
done